### PR TITLE
Properly translate and localize the bindings options view.

### PR DESCRIPTION
**Pull request by Web-eWorks**
(Finally!) added localization support to the new input system.

### CODE REVIEW

File: data/lang/input-core/en.json [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    },
+    "BIND_BOTTOM_CAMERA": {
+        "message": "Switch to Bottom Camera",
+        "description": "Descriptive name for the BottomCamera action."
+    },
+    "BIND_RESET_CAMERA": {
+        "message": "Reset Camera View to Default",
+        "description": "Descriptive name for the ResetCamera action."
+    },
+
```
Comment (impaktor) [inline data/lang/input-core/en.json]: Normally (as in all other files), we have description field before message field. I think @ecraven added something that will auto-format this when merged (confirm?), so no action should be needed, I think.

**Comment by impaktor** [comment]
Nice! 

Just a reminder to myself (or anyone else merging), I/we need to do some transifex magic to the new language resource, as usual.

**Comment by impaktor** [comment]
merge tomorrow?

**Comment by Web-eWorks** [comment]
Sounds good to me. I'm working on some control changes out of this branch, but they're mostly unrelated.

I can delete unused/unneeded keys from a resource without things breaking horribly, right?

**Comment by impaktor** [comment]
> I can delete unused/unneeded keys from a resource without things breaking horribly, right?

Yes

**Comment by impaktor** [comment]
Sorry for not getting around to this, but you can still make a new branch of this branch, and once this is merged, you can git cherry-pick the additional commits to a new PR.

**Comment by impaktor** [comment]
The reason I haven't merged this, is I'd like to know/remember how our json autoformater works. Not sure where that lives now, but I remember we have/had one? ping @ecraven

**Comment by impaktor** [comment]
I did an `amend` on your initial commit (jq formating of the json file), and pushed it back to this branch, if you wonder why commit hash / your branch has changed. 

Sorry for this taking so long, but took some time to get hold of ecraven for figuring out formating of the json file. Hope this hasn't stalled your tinkering with pioneer.

**Pull request merged**